.travis.yml

```diff
@@ -50,20 +51,24 @@ matrix:
       env:
         - MATRIX_EVAL="CXX=clang++-3.8"
           CXXFLAGS="-stdlib=libc++"
+          CXX_TYPE="clang"
```

Comment:
So for mpicxx this has to be set. But it does not need to be set here right?

Reply:
ah, I see. Probably not. It depends on wether our makefiles correctly figure out the CXX type from the CXX variable.